pytorch_tabnet/metrics.py

```diff
@@ -24,20 +24,26 @@ def UnsupervisedLoss(y_pred, embedded_x, obf_vars, eps=1e-9):
         Orginal input embedded by network
     obf_vars : torch.Tensor
         Binary mask for obfuscated variables.
-        1 means the variables was obfuscated so reconstruction is based on this.
+        1 means the variable was obfuscated so reconstruction is based on this.
     eps : float
         A small floating point to avoid ZeroDivisionError
         This can happen in degenerated case when a feature has only one value
 
     Returns
     -------
     loss : torch float
-        Unsupervised loss, average value over batch samples.    
+        Unsupervised loss, average value over batch samples.
     """
     errors = y_pred - embedded_x
-    reconstruction_errors = torch.mul(errors, obf_vars)**2
-    batch_stds = torch.std(embedded_x, dim=0)**2 + eps
+    reconstruction_errors = torch.mul(errors, obf_vars) ** 2
```

Comment:
We may call this `masked_vars`  instead of `obf_vars`

Reply:
`obf_vars` stands for obfuscated variables, I tried to make the distinction between obfuscated variables for self supervised training and masked variables which are masked by the attention layers. I think I prefer `obf_vars` but maybe it could be renamed `obfuscated_vars` for clarity.